### ISSUE TITLE
WordPress > Error - does not have a valid header

### ISSUE THREAD

**Issue by A-Buzon** [pods-framework/pods-beaver-builder-themer-add-on#96]
## Issue Overview
Cannot Activate plugin, gives WordPress Error "does not have a valid header".

## Expected Behavior
Cannot Activate plugin after install.

## Current Behavior
Search plugins via wp-admin-Add New.  Install.  Upon "Activate" screen goes to WordPress Error page, simply states "The plugin does not have a valid header."  URL of this error is:
wp-admin/plugins.php?_wpnonce=589c87413c&action=activate&plugin=pods-beaver-builder-themer-add-on/trunk/pods-beaver-themer.php

## Possible Solution
Three others have listed identical issues on Wordpress.org Support Forum for this plugin.

## Steps to Reproduce (for bugs)

1. WordPress 5.0.3 
2. Running Beaver Builder Child Theme theme with Beaver Themer and Pods already installed & activated.
3. Fresh build, just set up a few weeks ago.  All plugins & themes are current.  
4. Siteground hosting, running Wordfence and Smush but otherwise no other plugins, no caching, no CDN.

## Pods Package Export (helpful!)

```
Copy and Paste the JSON Export from **Pods Admin, Migrate: Packages, Export** in WordPress admin here
```

## Todos
- [ ] Tests
- [ ] Documentation

## WordPress Environment
<details>
```
Copy and paste your System Details from **Pods Admin > Settings > Debug Information** in WordPress admin here.
```
</details>

**Comment by quasel** [comment]
Weird - wasn't able to reproduce the issue :/

what's weird is your activation link - as there should be no /trunk/ - can you delete the plugin and do fresh download? Did you install it via admin or did you download / upload it ?

**Comment by quasel** [comment]
found the issue will release an update today - faulty SVN commit :/ weird that this happened only in rare times…

**Comment by quasel** [comment]
Fixed with #97 - and a new SVN commit - accidentally added a trunk folder with a second copy of the plugin! (version bump to get rid of it)
